authentik/stages/authenticator_validate/stage.py

```diff
@@ -218,6 +218,13 @@ def get_device_challenges(self) -> list[dict]:
                     "device_uid": device.pk,
                     "challenge": get_challenge_for_device(self.request, stage, device),
                 }
+                | (
```

Comment:
I think it'd be cleaner to have this in the same dict instead of using the `|` operator